src/main/java/com/purbon/kafka/topology/roles/rbac/RBACBindingsBuilder.java

```diff
@@ -354,7 +354,12 @@ public String toString() {
   @Override
   public List<TopologyAclBinding> setSchemaAuthorization(String principal, List<String> subjects) {
     return subjects.stream()
-        .map(subject -> apiClient.bind(principal, RESOURCE_OWNER).forSchemaSubject(subject).apply())
+        .map(
+            subject ->
+                apiClient
+                    .bind(principal, RESOURCE_OWNER)
+                    .forSchemaSubject(subject)
+                    .apply("Subject", subject))
```

Comment:
Hi @ludovic-boutros, thanks for taking care of this.

So if you're changing resource type from "Cluster" to "Subject" - do we still need a change in MDSApiClient.java ?
Seems like original condition shouldn't fire an issue anymore, right?

`binding.getResourceType().equals(ResourceType.CLUSTER.name())`

Reply:
Hi @lsolovey, yes we need to fix the client as well, because if you want to implement for instance _idempotence_ on producers for RBAC or any cluster level binding with resource you need this fix as well.

Reply:
@ludovic-boutros ok, I'm new to Kafka so I may not be aware of all the use cases.

Just curious - why _idempotence on producers for RBAC_ is significant here? Based on the Confluent doc, it's achieved by granting cluster-level `DeveloperWrite` or `ResourceOwner` role, without any resources-level RBAC bindings. 

https://docs.confluent.io/platform/current/security/rbac/rbac-cli-quickstart.html#produce-to-a-topic

Reply:
@lsolovey thanks for your answer. I had to implement the call for this MDS API in order to implement idempotence on producer for RBAC:
https://docs.confluent.io/platform/current/security/rbac/mds-api.html#post--security-1.0-principals-principal-roles-roleName-bindings

Reply:
both of you are correct, the way I kinda see this always done is by using DeveloperWrite/Read, not sure what do you mean @ludovic-boutros by the incremental API, this is anyhow a good addition and fix as it will certainly simplify the code and make all run better.